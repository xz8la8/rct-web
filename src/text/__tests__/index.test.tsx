import React from 'react';
import { mount, render } from 'enzyme';
import Text from '..';

describe('Text', () => {
  it('should render correctly', () => {
    const wrapper = mount(<Text />);
    expect(() => wrapper.unmount()).not.toThrow();
  });

  it('snapshot', () => {
    const wrapper = render(<Text />);

    expect(wrapper).toMatchSnapshot();
  })
});
