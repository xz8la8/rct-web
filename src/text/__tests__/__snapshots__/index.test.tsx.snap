// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text snapshot 1`] = `
<div>
  <h4>
    tag
  </h4>
</div>
`;
